Makefile

```diff
@@ -8,7 +8,7 @@ SDSL_DIR=../sdsl-lite
 #VERIFY_FLAGS=-DVERIFY_CONSTRUCTION
 
 # Multithreading with OpenMP and libstdc++ Parallel Mode.
-PARALLEL_FLAGS=-fopenmp -pthread -D_GLIBCXX_PARALLEL
+PARALLEL_FLAGS=-fopenmp -pthread
```

Comment:
`_GLIBCXX_PARALLEL` must be defined in a compiler flag (or at least before including any STL headers).

My long-term plan is to get rid of libstdc++ Parallel Mode (and OpenMP) completely. I just need a multithreaded in-place replacement for `std::sort` using C++ threads. After that, any compiler that can compile sdsl-lite should also be able to compile gcsa2.